infrastructure/main.tf

```diff
@@ -21,14 +28,45 @@ locals {
     "idam-prod"     = "production",
     "idam-aat"      = "staging",
     "idam-perftest" = "testing",
-    "idam-preview"  = "development"
+    "idam-preview"  = "development",
+    "idam-sandbox"  = "sandbox"
   }
   tags = merge(
     var.common_tags,
     {
-    "environment" = lookup(local.environments, var.env, replace(var.env, "idam-", ""))
+      "environment"         = lookup(local.environments, var.env)
+      "application"         = "cft-idam",
```

Comment:
I don't actually think we should be setting all those tags here - they are provided by default by the pipeline.
The only one that we need is 
```
environment"         = lookup(local.environments, var.env)
```

See https://github.com/hmcts/idam-api/blob/master/infrastructure/main.tf#L28-L33